README.md

```diff
@@ -32,11 +32,37 @@ We also have a [mailing list](https://lists.eff.org/mailman/listinfo/privacybadg
 This project uses the [QUnit](http://qunitjs.com/), [py.test](http://pytest.org/), [Selenium](http://www.seleniumhq.org/) test frameworks
 along with [Travis CI](https://travis-ci.org/) for continuous integration.
 
-To run unit tests, find your extension's ID (look for it on `chrome://extensions/`) and
-visit `chrome-extension://YOUR_EXTENSION_ID/tests/index.html`, replacing
-`YOUR_EXTENSION_ID` with your 32 character ID.
-
-To run functional tests powered by Selenium, you need to have `chromedriver`, `xvfb` and `python-virtualenv` installed. Also `geckodriver` to run functional tests in Firefox. See [scripts/setup_travis.sh](scripts/setup_travis.sh) and [tests/run_selenium_tests.sh](tests/run_selenium_tests.sh).
+#### Unit tests
+
+To run unit tests on Chrome find your extension's ID look for it on
+`chrome://extensions/`) and visit
+`chrome-extension://YOUR_EXTENSION_ID/tests/index.html`, replacing
+`YOUR_EXTENSION_ID` with your 32 character ID. For Firefox you can find your
+extensions's "UUID" on `about:debugging`, then visit
+`moz_extension://YOUR_EXTENSION_UUID/tests/index.html`.
+
+#### Functional tests
+
+To run the Selenium functional tests, you'll need to install `chromedriver` for
+Chrome or `geckodriver` for Firefox. Optionally you can also install `xvfb` to
+run the tests headlessly. You also need some python packages which can be installed by running:
+```bash
+$ pip install -r tests/requirements.txt
+```
+
+Now you should be able to run the selenium tests! Try running
+```bash
+$ BROWSER=chrome py.test
```

Comment:
And how to run without browser windows popping up. Maybe should instead default to headless and show a way to turn that off.